### ISSUE TITLE
What is the code binding method?

### ISSUE THREAD

**Issue by upup-721** [LianpoWang/MaskDICNet#20]
In which file is the code definition of the custom loss function in this repository located? What is the code binding method between the loss function and the optimizer during training?
